features/put_filter_outputs.feature

```diff
@@ -11,9 +11,9 @@ Feature: Put Filter Outputs Private Endpoints Enabled
         "filter_id": "94310d8d-72d6-492a-bc30-27584627fil1",
         "instance_id": "94310d8d-72d6-492a-bc30-27584627inst1",
         "state": "published",
-        "downloads": 
+        "downloads":
```

Comment:
These formatting commits, I believe, are from an earlier merge, and are being brought back in through this PR. I thought it was worthwhile as they are removing redundant spaces and tabs.

Reply:
Resolving as assuming this comment has been seen now.